lib/unleash/configuration.rb

```diff
@@ -63,7 +63,7 @@ def refresh_backup_file!
     end
 
     def fetch_toggles_url
-      self.url + '/client/features'
+      self.url + '/features'
```

Comment:
Actually the official path for clients to fetch feature toggles are `/client/features`. This was part of the v3 change of Unleash where the "api" and "client" paths was separated in order to allow different authentication mechanisms for clients and the admin ui. Support for the "/features" path directly on root is deprecated in `unleash-server` version 3.x and support for it will disappear in next version. (The server will always be backward compatible one version for clients). 

All version 3.x clients is supposed to fetch toggles via `/client/features`.

Reply:
@ivarconr Thank you for the explanation. We should fix GitLab side https://gitlab.com/gitlab-org/gitlab-ee/blob/master/ee/lib/api/unleash.rb#L22 indeed. 

Only my concern is that developer who was in charge of this feature tested with golang client, and there were no problems. So that golang client app might be still using deprecated `/features` path

Reply:
@ivarconr The Gitlab's implementation will be fixed in https://gitlab.com/gitlab-org/gitlab-ee/merge_requests/8045

Reply:
You are right, the go-client is indeed using the deprecated path. I raised an issue to solve this: https://github.com/Unleash/unleash-client-go/issues/30

Reply:
fyi: It is solved on the v3 branch: https://github.com/Unleash/unleash-client-go/blob/v3/repository.go#L66